api/src/main/java/org/pmiops/workbench/config/WorkbenchConfigMapper.java

```diff
@@ -72,9 +72,6 @@ default void mapRuntimeImages(WorkbenchConfig source, @MappingTarget ConfigRespo
   @Mapping(target = "enableGpu", source = "config.featureFlags.enableGpu")
   @Mapping(target = "enablePersistentDisk", source = "config.featureFlags.enablePersistentDisk")
   @Mapping(target = "enableMultiReview", source = "config.featureFlags.enableMultiReview")
-  @Mapping(
-      target = "enableDrugWildcardSearch",
-      source = "config.featureFlags.enableDrugWildcardSearch")
```

Comment:
Story only says to remove references to flag, but not the flag itself. Please revert.